### DISCUSSION TITLE
Before reinstalling windows do i need to reset the applied UNDERVOLT on the cpu ?

### DISCUSSION THREAD

**Discussion by RAGE-HERO7** [seerge/g-helper#1793] [Q&A] [answered]
* Is it safe to reinstall windows ignoring the applied UNDERVOLT to the cpu via GHELPER?

*Does the UNDERVOLT via GHELPER only get applied while we boot to windows or it stays UNDERVOLTED even after the device is power off ?

*Is it safe to install the chipset driver and igpu driver while the UNDERVOLT is applied and is in effect ?

* @seerge Sorry if i am asking too many questions, I love GHELPER, some of my friends have asus laptops so i want to make sure no serious inconvenience gonna get caused by GH while they use it on there laptop cause i want to install GH on there devices and potentially stop the problems causing by armory crate 🙂. My friends are still glued to the fancy design of armoury crate interface Lol😅

**Comment by seerge** [comment, marked as answer]
BIOS resets all settings after reboot even.
